contracts/rlp-decode.clar

```diff
@@ -3,67 +3,129 @@
 (define-constant ERR_INVALID_LENGTH (err u102))
 (define-constant MAX_SIZE 512)
 
-(define-private (rlp-decode-item (input (buff 512)))
-  (let ((first-byte (unwrap-panic (element-at? input u0))))
+(define-public  (decodeDeposit (input (buff 1024)))
```

Comment:
i think this should be in the balanced contracts so that this is a more generic rlp library